tools/seriesuid2fileid

```diff
@@ -1,4 +1,37 @@
 #!/usr/bin/perl
+
+=pod
+
+=head1 NAME
+
+seriesuid2fileid -- a script that displays relevant information and tracks the
+status of an acquired imaging series based on the series UID as an input
+argument (provided as a STDIN).
```

Comment:
Maybe rephrase the tiny description up there to something like:

"displays a report (including violation status) about imaging dataset(s) based on Series UID(s) given as STDIN"

Reply:
rephrased (mostly like you suggested with a little bit of difference); see latest commit and let me know if u disagree